### DISCUSSION TITLE
I want to change shapes of bar in bar charts as add little bit of radius.

### DISCUSSION THREAD

**Discussion by Dongseung** [nhn/tui.chart#721] [Q&A]
Hi, this is the first time of using tui.chart. I have searched and looked your api but I could not find any options or theme of changing shape of bar in barcharts. I need round shape of bar. Could you please let me know where I can find information or whether I can make it with options or not.

Thank you.

**Comment by jwlee1108** [comment]
@Dongseung 
I'm sorry for the late reply, We can't add `border-radius` to the bar chart.

**Reply by Dongseung** [reply]
Thank you for your reply.
